scanner/src/test/kotlin/ScanCodeTest.kt

```diff
@@ -19,12 +19,13 @@
 
 package com.here.ort.scanner
 
-import com.here.ort.model.CopyrightFindings
-import com.here.ort.model.LicenseFindings
+import com.here.ort.model.CopyrightFinding
```

Comment:
Commit message: "there need to be tests added FindingsMatcherTest" (I guess a "to" is missing?)

Reply:
done.